orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/WaitForDestroyedServerGroupTask.groovy

```diff
@@ -59,10 +59,12 @@ class WaitForDestroyedServerGroupTask extends AbstractCloudProviderAwareTask imp
       if (!cluster || !cluster.serverGroups) {
         return new DefaultTaskResult(ExecutionStatus.SUCCEEDED)
       }
-      if (!cluster.serverGroups.find { it.name == serverGroupName && it.region == serverGroupRegion }) {
+      def serverGroup = cluster.serverGroups.find { it.name == serverGroupName && it.region == serverGroupRegion }
+      if (!serverGroup) {
         return new DefaultTaskResult(ExecutionStatus.SUCCEEDED)
       }
-
+      def instances = serverGroup.instances?.findResults { it.name }?.join(',') ?: '[none]'
+      log.info("${serverGroupName}: not yet destroyed, found instances: ${instances}")
       return new DefaultTaskResult(ExecutionStatus.RUNNING)
```

Comment:
Any reason not to include the remaining instances in the context? ... might be easier to track down than trolling through log messages?